examples/example_pandas_train_test.py

```diff
@@ -0,0 +1,124 @@
+# -*- encoding: utf-8 -*-
+"""
+===============================
+Test and Train data with Pandas
+===============================
+
+*auto-sklearn* can automatically encode categorical columns using a label/ordinal encoder.
+This example highlights how to properly set the dtype in a DataFrame for this to happen,
+and showcase how to input also testing data to autosklearn.
+The X_train/y_train arguments to the fit function will be used to fit the scikit-learn model,
+whereas the X_test/y_test will be used to evaluate how good this scikit-learn model generalizes
+to unseen data (i.e. data not in X_train/y_train). Using test data is a good mechanism to measure
+if the trained model suffers from overfit, and more details can be found on `evaluating estimator
+performance <https://scikit-learn.org/stable/modules/cross_validation.html#cross-validation>`_.
+This example further highlights through a plot, the best individual models found by *auto-sklearn*
+through time (under indv_model_train_score/indv_model_test_score's legend). It also shows the
+training and test performance of the ensemble build using the best performing models (under
+ensemble_train_score and ensemble_test_score respectively).
+"""
+import time
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import sklearn.model_selection
+import sklearn.datasets
+import sklearn.metrics
+
+import autosklearn.classification
+from autosklearn.metrics import accuracy
+
+
+def get_runhistory_models_performance(automl):
+    metric = cls._automl[0]._metric
+    data = automl._automl[0].runhistory_.data
+    performance_list = []
+    for run_key, run_value in data.items():
+        if 'train_loss' not in run_value.additional_info:
```

Comment:
Would it be possible to check for SMAC's `StatusType` here?

Reply:
Yes, I will query for success then (all other cases like abort, crash and so on are not useful )